src/app/components/core/system-alert/system-alert.component.ts

```diff
@@ -15,15 +17,23 @@ export class SystemAlertComponent {
   expectedButtonLabels = ['Show Logs', 'Upgrade'];
   sortByKey = 'time';
   showSpinner = true;
+  isManualRefresh = false;
   public reenableButton = new EventEmitter<boolean>(false);
 
+  destroy$: Subject<boolean> = new Subject<boolean>();
   constructor(
     private alertService: AlertService,
     private router: Router,
+    private ping: PingService,
     private systemAlertService: SystemAlertService,
     public ngProgress: ProgressBarService,
     private rolesService: RolesService) {
-  }
+      this.ping.pingIntervalChanged
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((pingTime: number) => {
+        this.isManualRefresh = pingTime === -1 ? true : false;
+      });
```

Comment:
I think we can avoid this extra subscription of pingIntervalChanged here. We always have a active subscription for ping interval check in navbar. We can pass the information of isManualRefresh from the navbar as an Input() and use that information in ngOnChanges e.g. 
```
 if (changes?.alertsCount) {
      this.alertsCount = changes?.alertsCount?.currentValue;
    }
    if (changes?.isManualRefresh) {
      this.isManualRefresh = changes?.isManualRefresh?.currentValue;
    }